python/tk_multi_publish2/dialog.py

```diff
@@ -299,7 +308,7 @@ def _update_details_from_selection(self):
             context_item = self.ui.items_tree.topLevelItem(context_index)
             for child_index in xrange(context_item.childCount()):
                 child_item = context_item.child(child_index)
-                if child_item.enabled:
+                if child_item.checked:
```

Comment:
This was super confusing. The code was using `enabled` everywhere to indicate that a UI item was checked. Even more confusing is that we provided the ability for client code to enable/disable items meaning they can't be interacted with. I think I've fixed this throughout the code to make it clearer what's going on.

Reply:
ugh! thanks for fixing this!

Reply:
👍